cs2-dotnet-game/_Models/Sprites/Player.cs

```diff
@@ -0,0 +1,52 @@
+﻿using _Models.Sprites.Items;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _Models.Sprites;
+
+public class Player : Sprite
+{
+
+    private string name;
+    private int heathPoints;
+    private int spellPoints;
```

Comment:
missing Stamina points, and spellPoints could be renamed to just battle Points bc they are used by other attack types as well, but can also be added later